app/webpacker/css/admin_v3/components/tom_select.scss

```diff
@@ -54,3 +54,14 @@
     border-color: $orient;
   }
 }
+
+// For the "single" tom_select, be as clause as native select boxes
+// without too many options
+.ts-wrapper.single {
+  max-height: 40px;
```

Comment:
This height might better be defined with a relative `em` unit, in case we ever need to use a different font size.

Reply:
We probably also use a variable (as it's the same height as the buttons)

Reply:
see commits 0b486366a3 and 2742b26311

Reply:
Cool, if we need to remove 2px from the btn-height all the time, could the whole `calc` go inside `variables.scss`?

Reply:
It's use elsewhere without substracting 2px (the border of the button). That's why I prefer keeping this to 3em, therefore is the complete height of the button.

Reply:
Cool, I'll be quiet now, have a good day :)